src/GUI/Pages/Admin/User/CreateUser.razor.cs

```diff
@@ -39,19 +40,24 @@ private async Task HandleValidSubmitAsync()
             try
             {
                 await userService.CreateUserAsync(_userData);
-                UriHelper.NavigateTo("/");
+                _message = "Successfully created";
+                _isSuccessOperation = true;
+                StateHasChanged();
```

Comment:
Why be not moved StateHasShanged to method end?

Reply:
done